corehq/apps/userreports/templates/userreports/reportbuilder/configure_report.html

```diff
@@ -32,6 +32,7 @@
         var reportColumnOptions = {{ report_column_options|JSON }};
         var dateRangeOptions = {{ date_range_options|JSON }};
         var isGroupByRequired = {{ is_group_by_required|JSON }};
+        var groupByInitialValue = {{ form.fields.group_by.widget.value|JSON}};
```

Comment:
This widget's initial value was no longer being populated, so I added that in in this somewhat hacky fashion